wip ddm initial state

DiegoHu97 opened the pull request:
“”

pedroliveirav committed:
LRSD-4178 Identify cause of error in high priority contact forms (post-migration replay)

pedroliveirav committed:
LRSD-4178 Filter already selected contacts (post-migration replay)

pedroliveirav committed:
LRSD-4178 Treat error when there is a conflict in the high priority contact forms (post-migration replay)

Commit:
LRSD-4911 Create base test setup for master

eduardo-diniz committed:
LRSD-4769 Funcional Test - The fields never erase

Commit:
LRSD-4867 Create test for creating an MDF Request

Commit:
LRSD-4769 Added Fixture to Filters test

Commit:
LRSD-4867 SF

Commit:
LRSD-4911 SF

Commit:
LRSD-4911 Sort variables

Commit:
LPD-31144 Handle undo/redo events for rich text

gianmarcobrunialti committed:
LPD-29890 InfoBox tag (backward-compatible) enhancements for inner LFR modal handling

gianmarcobrunialti committed:
LPD-29890 `commerce-order-web` order detail modals adapted for enhanced `InfoBox` tag

gianmarcobrunialti committed:
LPD-29890 `commerce-order-web` order detail general page refactored to adopt enhanced `InfoBox`

jennyhchen committed:
LRSD-4178 Rename to update and add functions, sort, and format (post-migration replay)

Commit:
LRSD-4428 Change folder structure for class pages and tests

Commit:
LRSD-4428 Create page for MDF Request List

Commit:
LRSD-4428 Create tests for MDF Request List Page

Commit:
LRSD-4428 Create partnerHelper to handle partner requests

Commit:
LRSD-4428 Create test for search input

Commit:
LRSD-4428 Update parterHelper to create a properly account

Commit:
LRSD-4428 Create test for complete a MDF Request

jennyhchen committed:
LRSD-4178 General Formatting, rename associate functions and sort queries (post-migration replay)

kylembischof committed:
LRSD-5172 cc sales-ops on AC email (post-migration replay)

jennyhchen committed:
LRSD-4178 Simplify and add additional fix to InviteTeamMembers form (post-migration replay)

amosfong committed:
LRSD-5047 Reduce complimentary keys duration to 30 days (post-migration replay)

michelevigilante committed:
LPD-30000 Refactor Poshi tests

Commit:
LRSD-5100 Refactor MDFRequest Test Configs

Commit:
LRSD-5100 Create class for MDFRequestReview Page

Commit:
LRSD-5100 Refactor MDF Request Unit Tests to new approach

Commit:
LRSD-5100 Improve tests for MDFRequestForm

Commit:
LRSD-5100 Change approach to test MDFRequest Form Review

CharlotteFrancis committed:
LRCI-4098 add macrodef to build-test.xml to download gradle wrapper

CharlotteFrancis committed:
LRCI-4098 predownload the gradle wrapper in compile workspaces

CharlotteFrancis committed:
LRCI-4098 remove redundant logic in build-test-analytics-cloud.xml

MarinhoFeliphe committed:
LPD-32543 Define javaType attribute in Alias interface to dynamically pass its ORM Type

ryanschuhler committed:
LRSD-5100 Formatting and structure

gergelyszalay committed:
LPD-31694 Search in DL portlet does not show results in card view for LPS-202909

brbalazs committed:
LPD-32095 - Refactor of poshi test SFCommerceOrders#CanReturnOrdersBySearchForAccountName

Commit:
LRSD-5100 Refactor MDF Request tests to use partnerHelper

carolmariaabb committed:
LPD-20530 Add integration test to expose issue

carolmariaabb committed:
LPD-20530 Create the service context taking into account the object's entry status

CharlotteFrancis committed:
LRCI-4098 fix errors in download-gradle-wrapper-binary

carolmariaabb committed:
LPD-24914 Add unit test to expose issue

carolmariaabb committed:
LPD-24914 Prevent company-scoped object definitions from being selected when creating manual and dynamic collections.

Commit:
LPD-32220 Convert CheckWebBehaviorSegmentShowsWhichDataSourceDataCameFrom to playwright test

ryanschuhler committed:
LRSD-5100 Standardize type naming

ryanschuhler committed:
LRSD-5100 Update goto wait to be more performant

ryanschuhler committed:
LRSD-5100 More cleanup

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

brbalazs committed:
LPD-32095 - Deletion of poshi test SFCommerceOrders#CanReturnOrdersBySearchForAccountName

markocikos committed:
LPD-32489 Add PUT to the list of methods, simplify

markocikos committed:
LPD-32489 Add tests for available action form selections

ryanschuhler committed:
LRSD-5100 Change apiHelpers implementation

tinatian committed:
LPD-32543 SF, add TypeAlias to support the new feature

# breaking

## What modules/core/petra/petra-sql-dsl-api/src/main/java/com/liferay/petra/sql/dsl/expression/Expression.java

Add new method Expression#as(String, Class<?>)

## Why

This is needed to set specific return type for alias

----

tinatian committed:
LPD-32543 Apply usages

tinatian committed:
LPD-32543 Baseline

tinatian committed:
LPD-32543 Add test

Commit:
LPD-27774 Taken into account ctCollectionId during DDMFormInstanceSettingsUpgradeProcess v5_3_2 of the DDMFormInstance and DDMFormInstanceVersion entities

ryanschuhler committed:
LRSD-5100 Update login strategy

MarinhoFeliphe committed:
LPD-1901 Test formula object field with division operation

MarinhoFeliphe committed:
LPD-1901 Set Alias type based on the output object field setting to keep the decimals in arithmetric operations

pedro-oliveira446 committed:
LPD-32549 Update URL matching to account for potential friendly URLs like '~/control_panel/manage'

Commit:
LPD-27774 [Integration test] missing ctCollection Id: DDMFormInstance and DDMFormInstanceVersion

kyle-miho committed:
LPD-32555 Add gradle plugins test batch to test suite

pedro-oliveira446 committed:
LPD-32549 Integration Test

CharlotteFrancis committed:
LRCI-4098 SF

sandrodw3 committed:
LPD-32616 Replace Echo logo image

sandrodw3 committed:
LPD-31616 This projects does not exist anymore

sandrodw3 committed:
LPD-32616 Adapt method so it works without fields modal

sandrodw3 committed:
LPD-32616 The id is not needed

sandrodw3 committed:
LPD-32616 Use existing method

sandrodw3 committed:
LPD-32616 Use correct fixture names

sandrodw3 committed:
LPD-32616 Rename and improve method

DiegoHu97 committed:
LPD-31144 Save initial state for ddm fields

pedro-oliveira446 committed:
LPD-32549 Extract to private function

sandrodw3 committed:
LPD-32631 Don't consider relationship fields

sandrodw3 committed:
LPD-32631 Not needed

markocikos committed:
LPD-32489 Apply default fetch headers to all action requests

Commit:
LPD-32221 Set the same data-testid for all string type criteria

Commit:
LPD-32221 Update the editCriteriaAttributeValue function to be able to add criteria as nested if necessary

Commit:
LPD-32221 Update the dragAndDropCriteriaItem function to work in cases where a nested criterion is needed

Commit:
LPD-32221 Add the addNestedSegmentField function

Commit:
LPD-32221 Add the viewSegmentCriteriaCard function

Commit:
LPD-32221 Convert SegmentCriteriasNestCorrectlyInTheCriteriaCard to Playwright

Commit:
LPD-32221 SF

markocikos committed:
LPD-32489 Use `aria-label` for tab panes

markocikos committed:
LPD-32489 Remove Poshi tests covered in Playwright

devpedrotavares committed:
LPD-32664 Escape user name

gianmarcobrunialti committed:
LPD-30000 Improved test performance

Commit:
LPD-32511 - Escape rich text only when LPD-31212 feature flag is active

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

ryanschuhler committed:
LRSD-5100 More fixes

interaminense committed:
LPD-32706 Add props to hide info card and hide it on individuals tab

interaminense committed:
LPD-32706 Add unit test to cover info card scenario

CalumR23 committed:
LRCI-4409 Update to latest playwright image

CalumR23 committed:
LRCI-4409 Update package.json

smottal committed:
LPD-30530 Add ERC to roles (service)

smottal committed:
LPD-30530 Autogenerated Code (build-service-portal)

smottal committed:
LPD-30530 Update service layer

smottal committed:
LPD-30530 Upgrade process

andrea-ale-sbarra committed:
LPD-31973 added ERC column to Specification Group

gianmarcobrunialti committed:
LPD-29890 Changed locators

smottal committed:
LPD-30530 Semantic Versioning

# breaking

## What portal-kernel/src/com/liferay/portal/kernel/service/RoleLocalService.java

Add externalReferenceCode parameter on addRole method

## Why

To support external reference code for Role model

----

# breaking

## What portal-kernel/src/com/liferay/portal/kernel/service/RoleService.java

Add externalReferenceCode parameter on addRole method

## Why

To support external reference code for Role model

----

smottal committed:
LPD-30530 Update usage

andrea-ale-sbarra committed:
LPD-31973 change method signature

# breaking

## What modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPOptionCategoryLocalServiceImpl.java

Add and Update methods now accept ERC as parameter

## Why

The CPOptionCategory needs to hold value for ERC too

----

# breaking

## What modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPOptionCategoryServiceImpl.java

Add and Update methods now accept ERC as parameter

## Why

The CPOptionCategory needs to hold value for ERC too

----

andrea-ale-sbarra committed:
LPD-31973 adapt usage

ealonso committed:
LRCI-4421 Removes websphere from tests since it is not supported

drewbrokke committed:
LPD-32780 project-templates-api: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-api: uses new project template

drewbrokke committed:
LPD-32780 project-templates-control-menu-entry: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-control-menu-entry: uses new project template

drewbrokke committed:
LPD-32780 project-templates-form-field: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-form-field: uses new project template

drewbrokke committed:
LPD-32780 project-templates-fragment: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-fragment: uses new project template

drewbrokke committed:
LPD-32780 project-templates-layout-template: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-layout-template: uses new project template

drewbrokke committed:
LPD-32780 project-templates-mvc-portlet: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-mvc-portlet: uses new project template

drewbrokke committed:
LPD-32780 project-templates-npm-angular-portlet: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-npm-angular-portlet: uses new project template

drewbrokke committed:
LPD-32780 project-templates-npm-react-portlet: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-npm-react-portlet: uses new project template

drewbrokke committed:
LPD-32780 project-templates-npm-vuejs-portlet: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-npm-vuejs-portlet: uses new project template

drewbrokke committed:
LPD-32780 project-templates-panel-app: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-panel-app: uses new project template

drewbrokke committed:
LPD-32780 project-templates-portlet-configuration-icon: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-portlet-configuration-icon: uses new project template

drewbrokke committed:
LPD-32780 project-templates-portlet-provider: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-portlet-provider: uses new project template

drewbrokke committed:
LPD-32780 project-templates-portlet-toolbar-contributor: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-rest: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-rest: uses new project template

drewbrokke committed:
LPD-32780 project-templates-service-builder: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-service-builder: uses new project template

drewbrokke committed:
LPD-32780 project-templates-service-wrapper: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-service-wrapper: uses new project template

drewbrokke committed:
LPD-32780 project-templates-service: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-service: uses new project template

drewbrokke committed:
LPD-32780 project-templates-simulation-panel-entry: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-simulation-panel-entry: uses new project template

drewbrokke committed:
LPD-32780 project-templates-spring-mvc-portlet: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-spring-mvc-portlet: uses new project template

drewbrokke committed:
LPD-32780 project-templates-template-context-contributor: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-template-context-contributor: uses new project template

drewbrokke committed:
LPD-32780 project-templates-theme-contributor: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-theme-contributor: uses new project template

drewbrokke committed:
LPD-32780 project-templates-theme: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-theme: uses new project template

drewbrokke committed:
LPD-32780 project-templates-war-hook: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-war-hook: uses new project template

drewbrokke committed:
LPD-32780 project-templates-war-mvc-portlet: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-war-mvc-portlet: uses new project template

drewbrokke committed:
LPD-32780 project-templates-workspace: bnd.bnd: allows greater compatible version range

drewbrokke committed:
LPD-32780 project-templates-workspace: uses new project template

andrea-ale-sbarra committed:
LPD-31973 semantic versioning (gw buildService)

andrea-ale-sbarra committed:
LPD-31973 autogenerated code (gw buildService)

kyle-miho committed:
LPD-32555 Reenable failing gradle plugins tests

carolmariaabb committed:
LPD-20530 Fix integration test

drewbrokke committed:
LPD-32780 modules: bumps com.liferay.ant.bnd.version for project templates

drewbrokke committed:
LPD-32780 project-templates-portlet-toolbar-contributor: uses new project template

ling-alan-huang committed:
LPD-32570 SF, move variable declaration

ling-alan-huang committed:
LPD-32570 SF, move variable declaration

Commit:
LPD-32585 Add site-management test suite

pedro-oliveira446 committed:
LPD-2607 Validate null pointer

drewbrokke committed:
LPD-32780 project-templates: fixes NPE in test

devpedrotavares committed:
LPD-32664 Assert that scripts in user name don't get executed when viewing workflow tasks

javalosjr96 committed:
LPD-32711 unify the upgrade components first

pedro-oliveira446 committed:
LPD-2607 Add terms to prevent errors during evaluation

pedro-oliveira446 committed:
LPD-2607 Integration test

Commit:
LRSD-5100 Create New Mocks to MDF Request Data

Commit:
LRSD-5100 Update logic to use admin user

brianchandotcom committed:
LPD-32549 SF

brianchandotcom committed:
LPD-32549 SF

javalosjr96 committed:
LPD-32711 add component to relevant testcases

The portal upgrades component is being used for all upgrade testcases.  We should make a new one to keep them seperate for database components.

brianchandotcom committed:
LPD-32780 Temp revert while publishing

smottal committed:
LPD-30530 Data factory

brianchandotcom committed:
LPD-32780 prep next

shuyangzhou committed:
LPD-32689 Fix LibraryReferenceTest

javalosjr96 committed:
LPD-32711 add new component to test.props

jeffwu0724 committed:
LPD-32853 changed to use log4j2 with version 2.17.1.LIFERAY-PATCHED-1 in these wars

brittneyq committed:
LRCI-4420 Add service builder rule and add missing rules to list

michaelhashimoto committed:
LRCI-4416 Fix finding the Liferay & OSGi Logs sent to Testray

michaelhashimoto committed:
LRCI-4416 Make addTestClassMethod into a public method

michaelhashimoto committed:
LRCI-4416 Avoid adding TestClasses twice when using JSONObjects

michaelhashimoto committed:
LRCI-4416 Move playwright methods into a new class

kenjiheigel committed:
LRCI-4403 Fix typo

kenjiheigel committed:
LRCI-4403 Add uncommitted files

brianchandotcom committed:
LPD-32543 merge

brianchandotcom committed:
LPD-32543 auto SF

ealonso committed:
LPD-32860 Removes not needed component name, also all accessibility tests where migrated from poshi to playwright, so accessibility routine is not necessary

ealonso committed:
LPD-32860 We don't have any poshi tests with Page Audit component so remove it, also remove echo-analytics-cloud since it is not necessary anymore

ealonso committed:
LPD-32860 Upgrades WEM poshi tests where migrated to integration tests, so the routine and component name are not necessary

ealonso committed:
LPD-32860 Rename from Echo to Page Management

ealonso committed:
LPD-32860 Upgrades WEM poshi tests where migrated to integration tests, so the routine and component name are not necessary

ealonso committed:
LPD-32860 Rename from experience-management to page-management

ealonso committed:
LPD-32860 Rename from portal-wcm to portal-page-management

ling-alan-huang committed:
LPD-31151 Check all test.properties when we have test.properties removed in the current branch

ealonso committed:
LPD-32860 Removes no needed routine

ling-alan-huang committed:
LRSD-5435 Auto SF

ling-alan-huang committed:
LPD-31271 Auto SF

ling-alan-huang committed:
LPD-32549 Auto SF

dantewang committed:
LPD-32148 SF, sort parameters by name

dantewang committed:
LPD-32148 SF, use method reference

juanjofgliferay committed:
LPD-32879 - Feat. Update model to setup entryTitleFieldname so it could be displayed to select a Data Set in the fragment

juanjofgliferay committed:
LPD-32879 - Refactor. Send selected item label as title to itemSelector

juanjofgliferay committed:
LPD-32879 - Adjust button locator

juanjofgliferay committed:
LPD-32879 - Remove Poshi test

DiegoHu97 committed:
LPD-30408 Add history button

ealonso committed:
LPD-32860 Rename from Echo to Page Management

monaco-fabio committed:
LPD-31434 Remove all h4, h5 and h6 in DXP - Commerce

Commit:
LPD-32221 Update snapshots

michaelhashimoto committed:
LRCI-4421 Remove websphere batches from build-test-batch.xml

ambrinchaudhary committed:
LPD-30408 This styles are not needed

ambrinchaudhary committed:
LPD-30408 Add functionality to Undo All option

brianchandotcom committed:
LPD-30000 Rename

brianchandotcom committed:
LPD-30000 SF

brianchandotcom committed:
LPD-27774 Sort

ling-alan-huang committed:
LPD-32570 SF, move variable declaration

ling-alan-huang committed:
LPD-32570 SF, move variable declaration

brianchandotcom committed:
LPD-30530 SF

ling-alan-huang committed:
LPD-32570 SF, move variable declaration

ling-alan-huang committed:
LPD-32570 SF, move variable declaration

ling-alan-huang committed:
LPD-32570 SF, move variable declaration

ling-alan-huang committed:
LPD-32570 SF, move variable declaration

ling-alan-huang committed:
LPD-32570 SF, move variable declaration

brianchandotcom committed:
LRCI-4416 prep next

ling-alan-huang committed:
LPD-32570 SF

shuyangzhou committed:
LPD-30222 Company is checked upon creation, no need to re-check it here. This unnecessary check will lead to re-registering the company through PortalInstanceLifecycleListenerManagerImpl which will eventually lead to duplicated runs of SiteInitializers.

michaelhashimoto committed:
LRCI-4419 Add method getDuration to all TestrayCaseResult objects

CharlotteFrancis committed:
LRCI-4089 SF beanshell

michaelhashimoto committed:
LRCI-4419 Report testray.testcase.duration to all Testray imports

shuyangzhou committed:
LPD-30222 Per request invocation tracking for BundleSiteInitializer to skip duplicated calls within the same request renderring

ryanschuhler committed:
LRSD-5100 More cleanup

pei-jung committed:
LPD-32726 Get all missing requirements conflicts

shuyangzhou committed:
LPD-32912 AdvancedPermissionChecker is in portal-impl, no need to go through service for Resource creation.

pei-jung committed:
LPD-32726 Add test to replicate the issue

kevhlee committed:
LPD-32909 Add Japanese link to documentation

javalosjr96 committed:
LPD-32711 update our team name from upgrades to database infrastructure

ling-alan-huang committed:
LPD-32570 SF, move variable declaration

ling-alan-huang committed:
LPD-32570 SF, move variable declaration

ling-alan-huang committed:
LPD-32570 SF, move variable declaration

ling-alan-huang committed:
LPD-32570 SF, move variable declaration

ling-alan-huang committed:
LPD-32570 SF, move variable declaration

brianchandotcom committed:
LPD-32726 SF

brianchandotcom committed:
LRCI-4419 prep next

brianchandotcom committed:
LPD-32570 SF

brianchandotcom committed:
LPD-32570 SF

ealonso committed:
LPD-30530 Fix JSONRoleAPI.macro adding the externalReferenceCode to add-role api call

brianchandotcom committed:
LPD-32711 #sfme
